lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -1069,14 +1080,15 @@ def install_puppet_agent_pe_promoted_repo_on( hosts, opts )
         # to other settings whether the 32 or 64bit install is used
         #
         # @example
-        #   install_puppet_agent_dev_repo_on(host, { :sha => 'd3377feaeac173aada3a2c2cedd141eb610960a7', :version => '1.1.1.225.gd3377fe'  })
+        #   install_puppet_agent_dev_repo_on(host, { :sha => 'd3377feaeac173aada3a2c2cedd141eb610960a7', :puppet_agent_version => '1.1.1.225.gd3377fe'  })
         #
         # @return nil
         def install_puppet_agent_dev_repo_on( hosts, opts )
+          opts[:puppet_agent_version] ||= opts[:version] #backward compatability
           block_on hosts do |host|
             variant, version, arch, codename = host['platform'].to_array
             opts = FOSS_DEFAULT_DOWNLOAD_URLS.merge(opts)
-            opts[:download_url] = "#{opts[:dev_builds_url]}/puppet-agent/#{opts[:sha]}/repos/"
+            opts[:download_url] = "#{opts[:dev_builds_url]}/puppet-agent/#{ opts[:puppet_agent_sha] || opts[:puppet_agent_repo] }/repos/"
```

Comment:
`opts[:puppet_agent_repos]`? I don't see that value described in the docs. Do you mean `opts[:puppet_agent_version]`?